pixi.toml

```diff
@@ -167,6 +167,11 @@ man = "cargo --quiet run --package rerun-cli --all-features -- man > docs/conten
 # You can also give an argument for what to view (e.g. an .rrd file).
 rerun = "cargo run --package rerun-cli --no-default-features --features map_view,nasm,native_viewer --"
 
+# Compile and run the rerun viewer, with performance telemetry.
+#
+# You can also give an argument for what to view (e.g. an .rrd file).
+rerun-perf = "cargo run --package rerun-cli --no-default-features --features map_view,nasm,native_viewer,perf_telemetry --"
```

Comment:
when would I ever want to use debug for a build with performance telemetry? Seems like a footgun to me - I would have expected `rerun-perf` already to be release
Any objections to making all -perf variants be release and kill the dedicated release ones?

Reply:
I've used them plenty of times for debugging purposes (distributed tracing can be used as a debugger as much as a profiler, even Tracy can to some extent) -- many things we do that relate to Redap don't really need any compute on the client-side either, it's all IO-bound.

The core of the issue is that "performance telemetry" is really a misnomer, but I didn't want to call it just "telemetry" so people woudn't think this is some kind of analytics thing.

I can offer to switch it around: `rerun-perf` becomes release by default, and I'll keep `rerun-perf-debug` around (it really is a lifesaver when you're 200 datafusion compilations into your day...)